docusaurus/docs/iOS/uikit/data-formatting.md

```diff
@@ -76,3 +76,49 @@ Appearance.default.formatters.channelName = CustomChannelNameFormatter()
 ```
 
 In the example above, if the channel has only the current user as a member, it uses the name of the current user as the channel name, otherwise, the default logic is used.
+
+## Markdown
+
+The SDK offers Markdown formatting out of the box for channel messages. It supports the most common Markdown syntax; _italic_, **bold**, ~~strikethrough~~, `code`, Headings, Links, etc. It uses [SwiftyMarkdown](https://github.com/SimonFairbairn/SwiftyMarkdown) library internally.
+
+<center><img src={require("../assets/markdown-formatting.png").default} width="60%" height="60%"/></center>
+
+Mardown support is enabled by default. You can disable it by setting the following flag to false in the `Appearance` config:
+
+```swift
+Appearance.default.formatters.markdownFormatterEnabled = false
```

Comment:
Looking at the API right now, by the Apple's code guidelines for `Bool` types we could rename the flag to:
`isMarkdownEnabled`

Reply:
That makes more sense. Adjusted.